server/routes/chat_routes.ts

```diff
@@ -144,29 +145,25 @@ export function registerChatRoutes(router: IRouter, routeOptions: RoutesOptions)
       request,
       response
     ): Promise<IOpenSearchDashboardsResponse<HttpResponsePayload | ResponseError>> => {
-      if (!routeOptions.rootAgentId) {
-        context.assistant_plugin.logger.error(AgentIdNotFoundError);
-        return response.custom({ statusCode: 400, body: AgentIdNotFoundError });
+      if (!routeOptions.rootAgentName) {
+        context.assistant_plugin.logger.error(AgentNameNotFoundError);
+        return response.custom({ statusCode: 400, body: AgentNameNotFoundError });
       }
       const { messages = [], input, sessionId: sessionIdInRequestBody } = request.body;
       const storageService = createStorageService(context);
-      const chatService = createChatService();
+      const chatService = createChatService(context);
```

Comment:
Since we've already check `routeOptions.rootAgentName` is not empty, we can just pass it here.